cli/src/main/kotlin/OrtMain.kt

```diff
@@ -68,7 +67,7 @@ fun main(args: Array<String>) {
     exitProcess(0)
 }
 
-class OrtMain : CliktCommand(name = ORT_NAME, invokeWithoutSubcommand = true) {
+class OrtMain : CliktCommand(name = ORT_NAME, epilog = "* denotes required options.", invokeWithoutSubcommand = true) {
```

Comment:
What do you think about extracting a constant for `REQUIRED_OPTION_MARKER`, so it gets harder to break? e.g. also used in line 105